lms/djangoapps/certificates/models.py

```diff
@@ -339,9 +339,17 @@ def is_valid(self):
     def save(self, *args, **kwargs):
         """
         After the base save() method finishes, fire the COURSE_CERT_AWARDED
-        signal iff we are saving a record of a learner passing the course.
+        signal if we are saving a record of a learner passing the course.
```

Comment:
iff = if and only if, probably not a typo

Reply:
Huh, TIL